config/routes.rb

```diff
@@ -290,7 +290,7 @@
 
           resources :immunity_enforcements, only: :update
 
-          resource :considerations, only: %i[show edit update] do
+          resource :considerations, only: %i[edit update] do
```

Comment:
Is there a controller method that can also be deleted as a result of this?

Reply:
Already removed in a previous PR so this should have been captured then